libobs/obs-video.c

```diff
@@ -472,12 +472,8 @@ static inline void copy_rgbx_frame(
 {
 	uint8_t *in_ptr = input->data[0];
 	uint8_t *out_ptr = output->data[0];
-
-	for (size_t y = 0; y < info->height; y++) {
-		memcpy(out_ptr, in_ptr, info->width * 4);
-		in_ptr += input->linesize[0];
-		out_ptr += output->linesize[0];
-	}
+	
+	memcpy(out_ptr, in_ptr, info->width * info->height * 4);
```

Comment:
It wasn't very clear since it's adding ```output->linesize[0];``` and copy ```info->width * 4``` (which I guess are the same values).

Oh, and thanks for letting me know through IRC that I should not PR again ;)